README.md

```diff
@@ -36,142 +33,92 @@ If you don't yet have the code on your network you can clone this repo to your f
 
 ### Install Nextflow
 
-First, you need to be able to run nextflow.  Fortunately, it is easy to install:  
-[with details here] (<https://www.nextflow.io/),>  
-but if you already have java 1.8 installed you can just cd to the folder where you want  
-to install it and run  
-`curl -s https://get.nextflow.io | bash`.  
```

Comment:
This line seems to get mangled in the github rich diff.  Does it look ok on your side?